client/src/components/mapping/FhirMappingPanel/FhirResourceTree/index.tsx

```diff
@@ -111,7 +113,9 @@ const FhirResourceTree = ({ onClickCallback }: Props) => {
       id: attribute.tail,
       secondaryLabel: secondaryLabel(attribute, childNodes),
       nodeData: attribute,
-      label: ''
+      label: '',
+      isSelected:
+        !!selectedAttribute && attribute.path == selectedAttribute.path
```

Comment:
Why do we need that only now?

Reply:
we don't need that only now, but i just noticed now that it's disturbing that when you refresh the page, the "selectedAttribute" from the redux store is not selected in the FhirResourceTree.